packages/office-ui-fabric-react/src/components/Text/docs/TextDos.md

```diff
@@ -0,0 +1 @@
+- Use Text to get styled text out of the box.
```

Comment:
Thank you for adding Dos and Dont's! But, this particular comment doesn't seem very clear/actionable--maybe think of a way to rephrase it? Here's one idea--just riffing:
> Use Text to display read-only text that inherits styling from `ITheme`.

Reply:
Good call, it could definitely have better wording. 

I think your suggestion is great! I'll change it to that.